apps/openmw/mwrender/animation.cpp

```diff
@@ -1106,7 +1183,25 @@ namespace MWRender
         int mLowestUnusedTexUnit;
     };
 
-    void Animation::addGlow(osg::ref_ptr<osg::Node> node, osg::Vec4f glowColor)
+    void Animation::addSpellCastGlow(const ESM::MagicEffect *effect){
+
+        osg::Vec4f glowColor(1,1,1,1);
+        glowColor.x() = effect->mData.mRed / 255.f;
+        glowColor.y() = effect->mData.mGreen / 255.f;
+        glowColor.z() = effect->mData.mBlue / 255.f;
+
+        if (!mObjectRoot->getUpdateCallback()) // If there is no glow on object
+            addGlow(mObjectRoot, glowColor, 1.5); // Glow length measured from in-game as about 1.5 seconds
+
+        else if (dynamic_cast <GlowUpdater*>(mObjectRoot->getUpdateCallback())->isDone() == true) // If there was a temporary glow on object and it finished
```

Comment:
OK, I'm storing the pointer again, as I did in an earlier implementation in this PR. For this, though, as I mentioned above I moved glowUpdater into the mwrender namespace so that I can avoid a slew of warnings on the travis build about using the anonymous namespace in the header file, where I declare the glowupdater that I store the pointer in (I'll be watching the travis builds just to make sure).